examples/guestbook/README.md

```diff
@@ -146,10 +144,11 @@ redis-master            name=redis-master                         name=redis-mas
 This will cause all pods to see the redis master apparently running on <ip>:6379.  The traffic flow from slaves to masters can be described in two steps, like so.
 
 - A *redis slave* will connect to "port" on the *redis master service*
-- Traffic will be forwarded from the service "port" (on the service node) to the  *targetPort* on the pod which (a node the service listens to).
+- Traffic will be forwarded from the service "port" (on the service node) to the  *targetPort* on the pod that the service listens to.
 
 Thus, once created, the service proxy on each minion is configured to set up a proxy on the specified port (in this case port 6379).
 
+*targetPort* will default to *port* if it is omitted in the configuration. For simplicity's sake, we omit it in the following configurations.
```

Comment:
You say "following configurations" here, but you omit it in the previous one as well. Maybe just don't say "following."